packages/core-components/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -56,9 +58,8 @@ const useIconStyles = makeStyles(
       color: theme.palette.secondary.main,
     },
     label: {
-      fontSize: '0.7rem',
       textTransform: 'uppercase',
-      fontWeight: 600,
+      fontWeight: theme.typography.fontWeightBold,
       letterSpacing: 1.2,
```

Comment:
Those 100 font weight do make a difference huh, how do we deal with this one? 600 was not a standard font weight to begin with, at least from the theme...

Reply:
Btw the icons are different for source, don't think I changed that, I'm guessing you were comparing master & my branch which was somewhat behind?

Reply:
These are the standard font weights, what would you prefer, up to 700 like I did, down to 500 medium, or revert to non-standard 600?
![Screen Shot 2022-11-28 at 5 03 59 PM](https://user-images.githubusercontent.com/9698639/204390674-1e7e931d-b441-46fa-8dd0-331bcbd2f79d.png)

Reply:
I'm not picky. Just saying that the outcome should ideally look pretty close to what was there before. I believe that the _only_ difference was in font size honestly. If you scale down the "after" image a few percent so it aligns with the "before" image, they are almost identical. So I am not sure that weight is what's at play here

Reply:
I was comparing against the demo site btw, regarding the icon difference.

Reply:
Anyway, you ok with leaving it with the bold font weight from the theme?

Reply:
Sure. But could the size be adjusted?

Reply:
(as in, the effective font-size, which i think is what's affecting things here)

Reply:
Ok took a better look, I removed the font size because it was not standard, by the theme the smallest standard size would be `0.75rem` instead of the `0.7rem` I removed, I'll change the `Typography` to use the `caption` variant.
![Screen Shot 2022-11-30 at 2 13 36 PM](https://user-images.githubusercontent.com/9698639/204888297-d5f8cf48-2256-4488-95ad-1dea58aafe53.png)

Reply:
Done, and fixed the conflicts, could you check again if the font size is closer now to the original?